src/url.rs

```diff
@@ -2,6 +2,8 @@ use std::ops::Range;
 
 use crate::scanner::Scanner;
 
+const QUOTES: &[char] = &['\'', '\"'];
```

Comment:
We can add more quotes here or even use a crate for that. Wanted to keep it simple for now to make refactoring easier in the future.